Marlin/src/HAL/HAL_STM32/timers.h

```diff
@@ -33,6 +33,10 @@
 #define hal_timer_t uint32_t
 #define HAL_TIMER_TYPE_MAX 0xFFFFFFFF // Timers can be 16 or 32 bit
 
+#define SWSERIAL_TIMER_PRIORITY 0
```

Comment:
timer priority 0 is too high. that's equal to systick. nothing should have that priority.
Increase this and stepper by one.

Reply:
I had used that because it was what was used before (by 3rd-party SoftwareSerial code), but it can change.